pyproject.toml

```diff
@@ -9,6 +9,23 @@ requires = [
 ]
 build-backend = 'setuptools.build_meta'
 
+[tool.cibuildwheel]
+build = ['cp38-*', 'cp39-*', 'cp310-*']
+skip = '*-win32 *-i686'
+test-requires = 'pytest'
+test-command = 'pytest --pyargs pyflct'
+
+[tool.cibuildwheel.linux]
+before-all = 'yum install -y fftw-devel; git clean -xfd -e wheelhouse'
+
+[tool.cibuildwheel.macos]
+before-all = 'brew install fftw; git clean -xfd -e wheelhouse'
+
+[tool.cibuildwheel.windows]
+before-all = '%CONDA%\Scripts\conda.exe install fftw && git clean -xfd -e wheelhouse'
```

Comment:
oh goodness. Is this binary compatible with manylinux2014 or whatever we are building against?

Reply:
Who knows, I don't that's for sure.